modules/images.py

```diff
@@ -374,7 +374,7 @@ def save_image(image, path, basename, seed=None, prompt=None, extension='png', i
     save_to_dirs = (grid and opts.grid_save_to_dirs) or (not grid and opts.save_to_dirs and not no_prompt)
 
     if save_to_dirs:
-        dirname = apply_filename_pattern(opts.directories_filename_pattern or "[prompt_words]", p, seed, prompt)
+        dirname = apply_filename_pattern(opts.directories_filename_pattern or "[prompt_words]", p, seed, prompt).strip('\\ /')

```

Comment:
I'm thinking it's preferable to strip leading or trailing spaces and slashes here in case some future pattern does return a blank and isn't handled, or if a user just messes up and adds those things to the pattern directly, there doesn't seem to be a downside to some additional stripping here?